### ISSUE TITLE
Bug: Accessibility > Form elements should have a visible label

### ISSUE THREAD

**Issue by ogi-m** [mattermost/focalboard#2228]
Ensures that every form element has a visible label and is not solely labeled using hidden labels, or the title or aria-describedby attributes ([more info](https://dequeuniversity.com/rules/axe/4.3/label-title-only?application=AxeChrome))

1. Board title
   - Element location: 
 `input[placeholder="Untitled\ board"]`
   - Element source: 
`<input class="Editable title" placeholder="Untitled board" title="Meeting Notes" spellcheck="true" value="Meeting Notes">`
2. View selector
   - Element location: 
 `input[placeholder="Untitled\ View"]`
   - Element source: 
`<input class="Editable undefined" placeholder="Untitled View" title="By type" spellcheck="true" value="By type">`
3. Board column headers
   - Element location: 
 `input[title="Ad\ Hoc"]`
   - Element source: 
`<input class="Editable undefined" placeholder="New Select" title="Ad Hoc" spellcheck="true" value="Ad Hoc">`
